package/kedro_viz/data_access/managers.py

```diff
@@ -278,6 +276,7 @@ def add_dataset(
                 layer=layer,
                 tags=set(),
                 dataset=obj,
+                stats=self.get_stats_for_data_node(_strip_transcoding(dataset_name)),
```

Comment:
I am not sure how different Transcoded datasets are compared to a regular dataset. The function get_stats_for_data_node pulls the data from the stats.json file for the key - `dataset_name`. In case of Transcoded dataset - `ingestion.int_typed_shuttles@pandas1` and` ingestion.int_typed_shuttles@pandas2`, it checks for the dataset name `ingestion.int_typed_shuttles` in stats.json.

Reply:
While the data stored in stats.json is via 2 hooks - `after_dataset_loaded` and `after_dataset_saved`

Reply:
yeah, in principal transcoded datasets are just saved and loaded differently but it's the same dataset. I was a bit confused. this is fine :)